administration/src/components/regions/RegionOverview.tsx

```diff
@@ -0,0 +1,102 @@
+import { Button, Card, H4, TextArea } from '@blueprintjs/core'
+import React, { ReactElement, useState } from 'react'
+import styled from 'styled-components'
+import { useUpdateDataPolicyMutation } from '../../generated/graphql'
+import { useAppToaster } from '../AppToaster'
+
+const Content = styled.div`
+  padding: 0 6rem;
+  width: 100%;
+  z-index: 0;
+  flex-grow: 1;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: center;
+`
+const Label = styled(H4)`
+  text-align: center;
+`
+const ButtonBar = styled(({ stickyTop: number, ...rest }) => <Card {...rest} />)<{ stickyTop: number }>`
+  width: 100%;
+  padding: 15px;
+  background: #fafafa;
+  position: sticky;
+  z-index: 1;
+  top: ${props => props.stickyTop}px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+
+  & button {
+    margin: 5px;
+  }
+`
+
+const CharCounter = styled.span<{ hasError: boolean }>`
```

Comment:
Use `$hasError` so that styled components does not forward the prop to the DOM element

```suggestion
const CharCounter = styled.span<{ $hasError: boolean }>`
```